src/game/common/gameengine.cpp

```diff
@@ -118,12 +118,15 @@ void GameEngine::Init(int argc, char *argv[])
         &xfer,
         "Data/INI/Default/Science.ini",
         "Data/INI/Science.ini");
+// These don't work for standalone yet
+#ifdef GAME_DLL
```

Comment:
This does not look right. This macro is intended to compile in `GAME_DLL` specific code. But `Init_Subsystem(g_theMultiplayerSettings` is not `GAME_DLL` specific code.

Reply:
Open a bug report and describe the problem and write that this hack is currently put here to avoid the crash. Then we can merge it.

Reply:
Done in #624